prog/bootstrap_rhizome.rb

```diff
@@ -32,8 +32,18 @@ def user
 
 # LogLevel VERBOSE logs user's key fingerprint on login. Needed to have a clear audit track of which key was using to log in.
 LogLevel VERBOSE
+
+# Terminate sessions with clients that cannot return packets rapidly.
+ClientAliveInterval 2
+ClientAliveCountMax 4
 SSHD_CONFIG
 
+  LOGIND_CONFIG = <<LOGIND
+[Login]
+KillOnlyUsers=rhizome
+KillUserProcesses=yes
```

Comment:
`KillUserProcesses` is supposed to default to `yes` for  non-root users, according to https://www.freedesktop.org/software/systemd/man/latest/logind.conf.html . I don't see a problem with including this, but either the that documentation isn't matching behavior, or it is matching and this isn't necessary.